RELEASE_NOTES.md

```diff
@@ -7,16 +7,11 @@ Example Note:
 Details about the thing that changed that needs to get included in the Release Notes in markdown.
 -->
 
-# knife-ec2 0.17.0 release notes:
-In this release we have added features for adding exisiting ENI to server using `--primary-eni option`. Also added option `--instance-initiated-shutdown-behavior` to stop or terminate the instance on shutdown. There are a couple of bug fixes and enhancement as well.
+# knife-ec2 0.18.0 release notes:
+In this release we have added features for tagging EC2 instances in AWS and Chef separately. Option `--aws-tag` is used for tagging node in AWS and option `--chef-tag` is used for tagging the node in Chef. Also options `--tag-node-in-chef` and `--tags` are deprecated from this release. There are a couple of bug fixes and enhancement as well.
```

Comment:
Since there is only one PR which is got merged in this release we should remove the last line 'There are a couple of bug fixes and enhancement as well' . @btm please correct me if I am wrong.

Reply:
Sure, I will do that right away.